### DISCUSSION TITLE
Rstudio Appearance in Positron

### DISCUSSION THREAD

**Discussion by seewe** [posit-dev/positron#5842] [Feedback] [answered]
I saw that Positron Actually use most of the color theme in VScode market place. Which is already great !
Please is it easy and possible to add default theme colors in Rstudio in Positron ? It will make developper familiar with the new interface ! Thank !

![image](https://github.com/user-attachments/assets/7112ef06-92e3-44c8-bcd4-7b21b0d97328)

**Comment by juliasilge** [comment, marked as answer]
We don't plan to exactly reproduce the themes from RStudio, given what a plethora of theme extensions are out there available for VS Code / Positron. If you are interested in something that looks very much like RStudio, you may want to look for [themes like this one](https://github.com/nanxstats/vscode-textmate-rstheme) that have been built to emulate RStudio!

**Comment by sebdalgarno** [comment]
The biggest issues I have in terms of getting used to the appearance of Positron (from RStudio) are lack of clear border separation between panels and lack of color differentiation in console text. Despite all the obvious improvements that Positron offers, it is really difficult for my brain to work around these and I end up moving back to RStudio.

**Reply by justinasR-19** [reply]
I have same issue. Please have a read this post and if you find it useful vote it up.

https://github.com/posit-dev/positron/discussions/5853

**Reply by juliasilge** [reply]
One of the great things about building on the VS Code infrastructure is that you can customize _a lot_ of this. If you are someone who likes more dramatic border separation between panels, you can do that via [user-level theming](https://code.visualstudio.com/api/references/theme-color). Look for items called "border" on that page, and you might try something like this in your `settings.json`:

```json
    "workbench.colorCustomizations": {
        "panel.border": "#333",
        "statusBar.border": "#333",
        "activityBar.border": "#333",
        "sideBar.border": "#333"      
    }
```

**Reply by justinasR-19** [reply]
Thank you for this. I found some online builders that I could have a go with to recreate COBALT theme from Rstudio. As per my initial post about the layout itself and spacing, Is it possible to adjust the spacing with this json file?  If not, do you have any internal discussions about window adjustments?

**Reply by sebdalgarno** [reply]
thanks @juliasilge I will give that a try!

**Reply by juliasilge** [reply]
@justinasR-19 I believe that VS Code / Positron themes are more about _color_ and not so much about spacing; AFAIK there aren't settings exposed for the aspects of the UI you outlined in https://github.com/posit-dev/positron/discussions/5853#discussioncomment-11753764. Thank you for your post outlining what you are missing coming from RStudio!

**Comment by gvelasq** [comment]
For those who are fans of RStudio's implementation of Tomorrow Night Bright, I have drafted a port of RStudio's Tomorrow Night Bright in the Positron editor with workbench decorations ported from GitHub Dark Default. I plan to release this extension on the VS Code Marketplace and Open VSX.

Repository: https://github.com/gvelasq/tomorrow-night-bright-r-classic

Related discussion: https://github.com/posit-dev/positron/discussions/3686#discussioncomment-12048757

**Reply by seewe** [reply]
Hi @gvelasq ! great what you did here ! Please can you replicate the same project for R classic Cobalt theme ? I'm actualy use the Twin Blue one ! But if it's easy for you to replicate the R classic Cobalt theme,  this will be great for me and some folks i know around me ! Thank !

**Reply by gvelasq** [reply]
The 'Tomorrow Night Bright (R Classic)' extension is now published.

Open VSX: https://open-vsx.org/extension/gvelasq/tomorrow-night-bright-r-classic
VS Code Extension Marketplace: https://marketplace.visualstudio.com/items?itemName=gvelasq.tomorrow-night-bright-r-classic

The scope for my extension is limited to a single theme, I don't plan to work on others (for now). I'm working with @ivelasq on a blog post that will walk people through the process which interested parties might apply to Cobalt. If you're interested please stay tuned for that! You're welcome to fork my repository and create your own as well.

**Reply by henrydehe** [reply]
> Hi @gvelasq ! great what you did here ! Please can you replicate the same project for R classic Cobalt theme ? I'm actualy use the Twin Blue one ! But if it's easy for you to replicate the R classic Cobalt theme, this will be great for me and some folks i know around me ! Thank !

+1
